### ISSUE TITLE
Multi-capabilities or multi-driver support

### ISSUE THREAD

**Issue by JeroMiya** [bbaia/protractor-net#23]
I'm a bit new to the C# selenium webdriver, so let me know if this is covered elsewhere that I'm not aware of.

Currently, it's currently a bit difficult to run e2e tests against multiple browsers, even when you're wrapping a RemoteWebDriver that points to a selenium grid which supports multiple browsers. Current solutions I'm aware of involve using app.config or environment variables/etc... to run tests multiple times and dynamically create drivers/capabilities based on config.

Protractor for JavaScript supports the concept of multiple capabilities, which allows you to easily specify multiple target browsers to run in parallel in one test run:
https://github.com/angular/protractor/blob/master/spec/multiConf.js

It would be nice to have this support for Protractor.net as well. I'm not sure if you'd have multi-capabilities like the JS version or just allow us to pass in an IEnumerable of web drivers to wrap, instead of just one. I suspect the multi-driver option might be easier to configure (and probably to implement).

**Comment by safrrhmn** [comment]
@JeroMiya  I believe this multi-browser concepts is handled by the test framework not this port. This can be implemented with NUnit, which is probaly the case here, which handles parametrized test and executing them. NUnit is pretty strong enough to handle the execution sequentially(if you want parallel testing, that can be done with Selenium Grid as you have mentioned). You may find [this](http://stackoverflow.com/questions/12164738/sequential-execution-of-webdriver-suite-c-sharp-selenium-grid-all-local) useful.

**Comment by bbaia** [comment]
Thanks @safrrhmn !
